src/serializer/ResidualHeapVisitor.js

```diff
@@ -73,11 +73,12 @@ export class ResidualHeapVisitor {
   modules: Modules;
 
   declarativeEnvironmentRecordsBindings: Map<DeclarativeEnvironmentRecord, VisitedBindings>;
+  // Either realmGenerator or the FunctionValue of an additional function to serialize
```

Comment:
Comment doesn't make sense here, and the realmGenerator is no more.

Reply:
Seems to belong to the code in line 83.